src/db/identity/database.ts

```diff
@@ -16,47 +16,47 @@ WHERE status COLLATE utf8_unicode_ci = 0
 GROUP BY authentication_id
 HAVING COUNT(*) > 1;`
 
-const DELETE_INVALID_TOKEN_SQL = `delete from token where status = 1 limit ${deleteBatchSize};`
-const DELETE_VALID_USER_TOKEN_SQL = `delete from token where status = 0 and user_name is not null limit ${deleteBatchSize};`
+const DELETE_INVALID_TOKEN_SQL = 'delete from token where status = 1 limit ?;'
+const DELETE_VALID_USER_TOKEN_SQL = 'delete from token where status = 0 and user_name is not null limit ?;'
```

Comment:
Looks fantastic! Maybe include the status field in your variables as well?

```diff
- where status = 1
+ where status = ?
```

Reply:
Don't think that's as necessary as we're not passing that in explicitly anywhere? It's already safe when hardcoded